autorelease/__main__.py

```diff
@@ -25,7 +25,7 @@
 def _determine_github_token(github_token):
     """Automatically use the GitHub token provided by Keystore if needed."""
     if github_token is not None:
-        return
+        return github_token
```

Comment:
I'm a bit confused as to why were weren't doing this before.

Reply:
The current execution flow doesn't pass the token from the command line. It's exercising the logic below this line to read from the keystore file location